lib/src/list_extensions.dart

```diff
@@ -286,6 +287,29 @@ extension ListExtensions<E> on List<E> {
       yield slice(i, min(i + length, this.length));
     }
   }
+
+  /// Returns a new list with the elements of [other] are removed from `this`.
```

Comment:
[nit] This returns an Iterable, not a list. Also the phrasing of "are removed" phrasing is awkward.

Maybe

/// Returns an iterable of the elements in this list that are not present in [other].

Reply:
Applied. I wanted to avoid 'present' as that somehow gives me the feeling that an occurrence in `[other]` results in removal of all same items in the result which deliberately isn't the case here (as we are aware about count and filter thing the amount they present in `[other]`) but I don't know to express that.

Also is it better to rename `[other]` to `[minuend]`? I renamed to `[other]` to make it more compatible with Set's `difference`.